### ISSUE TITLE
Assignment missing attribute

### ISSUE THREAD

**Issue by ddelblanco** [kstateome/canvas-api#146]
Hello:

The assignment object in canvas has an attribute that is not in the model (and curiously is not in the API documentation but it comes when you get assignments). 

The attribute name is "secure_params" and it is a JWT string, signed, that contains such interesting things like the ResourceLinkId, that can be used to call the LTI AGS Lineitems endpoint to get the lineitem(s) for that assignment. 

Basically the change will be adding this in the Assignment model:

```
private String secureParams;

    public String getSecureParams() {
        return secureParams;
    }

    public void setSecureParams(String secureParams) {
        this.secureParams = secureParams;
    }
```
